pkg/kubectl/cmd/version.go

```diff
@@ -46,36 +54,93 @@ func NewCmdVersion(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	}
 	cmd.Flags().BoolP("client", "c", false, "Client version only (no server required).")
 	cmd.Flags().BoolP("short", "", false, "Print just the version number.")
+	// default behavior is std.
+	cmd.Flags().String("output", "", "output format, options available are yaml and json")
 	cmd.Flags().MarkShorthandDeprecated("client", "please use --client instead.")
 	return cmd
 }
 
 func RunVersion(f cmdutil.Factory, out io.Writer, cmd *cobra.Command) error {
-	v := fmt.Sprintf("%#v", version.Get())
```

Comment:
That would be the ideal case. However, the "" case makes it a bit difficult because the output is actually `label: version.Info{...}` which makes this approach difficult to do. I have to keep the old output because legacy tests etc.. still use this.